package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "scripts": {
     "start": "npm-run-all -p serve:css serve",
-    "serve": "vite",
+    "serve": "vite --host",
```

Comment:
I dont think we should be doing this here as we `serve` is a subcommand used in the `start` command which we use for development. I think it would be better only exposing host in the CI start command to solve this issue. We can do this by changing [this command](https://github.com/sanger/traction-ui/blob/develop/.github/workflows/ci.yml#L83) to a custom one for cypress CI.

Reply:
I dont think having separate commands would make it inconsistent as they would be the same except one exposes to the network. Production env is also different as its bundled so it doesn't use these scripts.

Im not entirely sure what gets exposed to the network but the file system will be and im not sure thats something we want to risk if we are on internal networks etc.